src/components/component.ts

```diff
@@ -262,12 +262,14 @@ export class Component {
    * @param {String|d3.Selection} element A selector-string for the <svg>, or a d3 selection containing an <svg>.
    * @returns {Component} The calling Component.
    */
-  public renderTo(element: String | d3.Selection<void>): Component {
+  public renderTo(element: String | SVGSVGElement | d3.Selection<void>): Component {
```

Comment:
I prefer `HTMLElement` instead of `SVGSVGElement`

The reason for that is that in 99% of the cases the element is taken by using 
```javascript
document.getElementById("foo") // the type of this is not compatible with SVGSVGElement because it is too generic
```